### ISSUE TITLE
Increase server response time?

### ISSUE THREAD

**Issue by hedyt2000** [mdesilva/AmazonFlexUnlimited#14]
Hello
I have setup the app successfully, but its slow compared to other scripts. I also tried it on vps in ashburn with low latency but still it misses some blocks. In the getOffers I used the elapsed.time for post request and it is usually between 0.10-0.20 seconds. Can it be made any faster so that it get offers quickly and post the request successfully?. Also, when it misses the block the server response is 410. thanks

**Comment by mdesilva** [comment]
Are you using a wired Ethernet connection ? I found that wired really helps to speed up the acceptance of blocks over using a wireless connection. As for the 410 status code, that makes sense since 410 = Gone, and when it misses the block it is essentially "gone".

**Comment by hedyt2000** [comment]
Yeah wired connection is little bit faster than wireless but as compared to other scripts which charges money it is slow. I have used the other services and they successfully catches the blocks. I wonder what code they are using?

**Comment by figamore** [comment]
> Yeah wired connection is little bit faster than wireless but as compared to other scripts which charges money it is slow. I have used the other services and they successfully catches the blocks. I wonder what code they are using?

Can you provide an example of these other scripts? I may be able to analize what they're doing differently.

**Comment by hedyt2000** [comment]
They don't share their source code. The only thing I know is that, when you sign up for their services, a new device is added to your amazon account while using this script there was no device added to my amazon account.

**Comment by figamore** [comment]
I’m gonna sign up for the bot and run a proxy on it to see their headers. What’s the name of it? I’m really interested in making this repo as efficient as possible.

**Comment by hedyt2000** [comment]
www.flexninjas.com and www.flexernow.com. I have used proxy on them and their server ip address is in ashburn. I think their website is on that server not the actual script.

**Comment by sack2116** [comment]
> I’m gonna sign up for the bot and run a proxy on it to see their headers. What’s the name of it? I’m really interested in making this repo as efficient as possible.
> […](#)
> On Aug 14, 2022, at 8:47 PM, hedytt ***@***.***> wrote: ﻿ They don't share their source code. The only thing I know is that, when you sign up for their services, a new device is added to your amazon account while using this script there was no device added to my amazon account. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.

did you ever looked into this and figure out the best approach for this repo? im constantly getting 410 error code..

**Comment by luizsilva-dev** [comment]
I am also getting error code 410 constantly. Is there already a solution?
